examples/wifi/requests_wifi_status_codes.py

```diff
@@ -0,0 +1,150 @@
+
+# Get WiFi details, ensure these are setup in settings.toml
+ssid = os.getenv("CIRCUITPY_WIFI_SSID")
+password = os.getenv("CIRCUITPY_WIFI_PASSWORD")
+
+# Initalize Wifi, Socket Pool, Request Session
+pool = adafruit_connection_manager.get_radio_socketpool(wifi.radio)
+ssl_context = adafruit_connection_manager.get_radio_ssl_context(wifi.radio)
+requests = adafruit_requests.Session(pool, ssl_context)
+rssi = wifi.radio.ap_info.rssi
+
+
+def print_http_status(code, description):
+    """Returns HTTP status code and description"""
+    if "100" <= code <= "103":
+        print(f" | ✅ Status Test: {code} - {description}")
+    elif "200" <= code <= "299":
+        print(f" | ✅ Status Test: {code} - {description}")
+    elif "300" <= code <= "600":
+        print(f" | ❌ Status Test: {code} - {description}")
+    else:
+        print(f" | Unknown Response Status: {code} - {description}")
+
+
+# All HTTP Status Codes
+http_status_codes = {
+    "100": "Continue",
+    "101": "Switching Protocols",
+    "102": "Processing",
+    "103": "Early Hints",
+    "200": "OK",
+    "201": "Created",
+    "202": "Accepted",
+    "203": "Non-Authoritative Information",
+    "204": "No Content",
+    "205": "Reset Content",
+    "206": "Partial Content",
+    "207": "Multi-Status",
+    "208": "Already Reported",
+    "226": "IM Used",
+    "300": "Multiple Choices",
+    "301": "Moved Permanently",
+    "302": "Found",
+    "303": "See Other",
+    "304": "Not Modified",
+    "305": "Use Proxy",
+    "306": "Unused",
+    "307": "Temporary Redirect",
+    "308": "Permanent Redirect",
+    "400": "Bad Request",
+    "401": "Unauthorized",
+    "402": "Payment Required",
+    "403": "Forbidden",
+    "404": "Not Found",
+    "405": "Method Not Allowed",
+    "406": "Not Acceptable",
+    "407": "Proxy Authentication Required",
+    "408": "Request Timeout",
+    "409": "Conflict",
+    "410": "Gone",
+    "411": "Length Required",
+    "412": "Precondition Failed",
+    "413": "Payload Too Large",
+    "414": "URI Too Long",
+    "415": "Unsupported Media Type",
+    "416": "Range Not Satisfiable",
+    "417": "Expectation Failed",
+    "418": "I'm a teapot",
+    "421": "Misdirected Request",
+    "422": "Unprocessable Entity",
+    "423": "Locked",
+    "424": "Failed Dependency",
+    "425": "Too Early",
+    "426": "Upgrade Required",
+    "428": "Precondition Required",
+    "429": "Too Many Requests",
+    "431": "Request Header Fields Too Large",
+    "451": "Unavailable For Legal Reasons",
+    "500": "Internal Server Error",
+    "501": "Not Implemented",
+    "502": "Bad Gateway",
+    "503": "Service Unavailable",
+    "504": "Gateway Timeout",
+    "505": "HTTP Version Not Supported",
+    "506": "Variant Also Negotiates",
+    "507": "Insufficient Storage",
+    "508": "Loop Detected",
+    "510": "Not Extended",
+    "511": "Network Authentication Required",
+}
+
+JSON_GET_URL = "https://httpbin.org/get"
+STATUS_TEST = "https://httpbin.org/status/"
```

Comment:
The url itself is in JSON format so best to stick with the json formatting?  Might have gotten that one opposite as the JSON_GET_URL was original and STATUS_TEST was the new addition.  I've removed STATUS_TEST since that's been moved into its own example.  Returned this example to more of its original version except its now using a with statement and a few extra headers.

Reply:
I'm not sure if I understand what you mean about the URL being in JSON format. Do you mean the data that gets returned by the API at that URL? If so, I do think it'd be even more descriptive to include JSON in the variable name like `STATUS_TEST_JSON_URL`

This code that the comment chain here is attached to is the `requests_wifi_status_codes.py` file. Which is the one checking many status codes, not just the single 200 OK one. 

I was trying to say that I think the name of `STATUS_TEST` variable should be changed to `STATUS_TEST_URL` since the value of the variable is a URL this makes the code more understandable when reading it.

Reply:
@FoamyGuy Ohhh they're in the same commit.  Darn it I meant to separate status codes into a separate PR.  This makes more sense now.  Will do!